### PR TITLE
docs: add command menu

### PR DESCRIPTION
**Pull request by jim761**
Dialog width is fixed

### CODE REVIEW

**Review by aromko — CHANGES_REQUESTED** [review]
@jim761 I took a look at your PR. Feel free to ask if you have any questions.

File: docs/ui/Commandk.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }) => {
+      if (e.key === 'k' && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        setOpen(open => !open);
+      }
+    };
+    document.addEventListener('keydown', handleKeyDown);
+    return () => document.removeEventListener('keydown', handleKeyDown);
+  }, []);
+  const router = useRouter();
```
Comment (aromko) [inline docs/ui/Commandk.tsx:28]: Put these 3 consts before the useEffect. Than I'm happy :-)

File: docs/ui/Commandk.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  const [open, setOpen] = React.useState(false);
+  const [search, setSearch] = React.useState('');
+
+  const groupedPages = siteConfig.navigation.map(({ name, slug }) => {
+    const items = allContentPages
+      .filter(page => page.slug.includes(slug))
+      .map(({ title, slug, order }) => ({ title, slug, order }));
+    //sort by order or alphabetticly
+    for (let i = 0; i < items.length; i++) {
```
Comment (aromko) [inline docs/ui/Commandk.tsx:37]: There's more to it. Tip: You can write it in such a way that you don't have any nested for-loops.

**Review by aromko — CHANGES_REQUESTED** [review]

File: docs/ui/Commandk.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    };
+    document.addEventListener('keydown', handleKeyDown);
+    return () => document.removeEventListener('keydown', handleKeyDown);
+  }, []);
+
+  const groupedPages = siteConfig.navigation.map(({ name, slug }) => {
+    const items = allContentPages
+      .filter(page => page.slug.includes(slug))
+      .map(({ title, slug, order }) => ({ title, slug, order }));
+    //sort by order or alphabetticly
```
Comment (aromko) [inline docs/ui/Commandk.tsx:37]: I think you can remove some comments here. It's kinda doubled.

File: docs/ui/Commandk.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    //sort by order if there is an order
+    //if there is no order sort alphabetticly
+    items.sort((a, b) => {
+      if (typeof a.order == 'number' && typeof b.order == 'number') {
```
Comment (aromko) [inline docs/ui/Commandk.tsx:41]: Use [strict comparison](https://www.telerik.com/blogs/indepth-understanding-strict-abstract-equality-operators-javascript#:~:text=In%20JavaScript%2C%20the%20abstract%20and,not%20of%20the%20same%20type.). 

Than it's ok from my side. Well done 👍

**Review by aromko — CHANGES_REQUESTED** [review]

File: docs/app/themes.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+import '@marigold/theme-b2b/styles.css';
```
Comment (sebald) [inline docs/app/themes.ts:1]: Delete, because unused. Sorry I added that :D

Comment (jim761) [reply docs/app/themes.ts:1]: Delete the file?

Comment (sebald) [reply docs/app/themes.ts:1]: yep

File: docs/ui/Commandk.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    setOpen(false);
+  }
+
+  return (
+    <Dialog.Trigger open={open} onOpenChange={setOpen} dismissable>
+      <Button variant="sunken" size="small" onPress={() => setOpen(true)}>
+        <span className="hidden xl:inline-flex ">Search documentation...</span>
+        <span className="inline-flex xl:hidden ">Search...</span>
+        <div className="hidden h-5 w-10  items-center justify-center rounded-md text-sm lg:inline-flex lg:border lg:border-gray-300 lg:bg-gray-200">
+          <span className="text-xs ">⌘</span>K
```
Comment (sebald) [inline docs/ui/Commandk.tsx:60]: The CMD icon might be weird for non-Mac users. Switch between CMD/Ctrl based on the OS?

**Review by aromko — APPROVED** [review]

File: docs/ui/Commandk.tsx (hunk 6 above)
Comment (sebald) [reply docs/ui/Commandk.tsx:60]: Also should be a "kbd" not a span :)

**Comment by sebald** [comment]
Set to draft, we want to include this in the release after the next one :D

**Review by sebald — APPROVED** [review]

**Pull request merged**